syntax = "proto3";

import "EmptyAcknowledge.proto";
import "HelloMessages.proto";
import "google/protobuf/any.proto";

service Router
{
    rpc SayHello (HelloRequest) returns (HelloReply) {}
    rpc SendCommand (google.protobuf.Any) returns (Messages.EmptyAcknowledge) {}
}